src/app/Fake.DotNet.Testing.SpecFlow/SpecFlow.fs

```diff
@@ -0,0 +1,85 @@
+/// Contains a task which allows to run [SpecFlow](http://www.specflow.org/) tests.
+[<RequireQualifiedAccess>]
+module Fake.DotNet.Testing.SpecFlow
+
+open Fake.Core
+open Fake.IO
+open Fake.IO.Globbing
+open Fake.IO.FileSystemOperators
+open System.IO
+open System.Text
+
+/// SpecFlow execution parameter type.
+[<CLIMutable>]
+type SpecFlowParams = { 
+    SubCommand:         string
+    ProjectFile:        string
+    ToolName:           string
+    ToolPath:           string
+    WorkingDir:         string
+    BinFolder:          string
+    OutputFile:         string
+    XmlTestResultFile:  string
+    TestOutputFile:     string
+    Verbose:            bool
+    ForceRegeneration:  bool
+    XsltFile:           string
+}
+
+let private toolname = "specflow.exe"
+let private currentDirectory = Directory.GetCurrentDirectory ()
+
+/// SpecFlow default execution parameters.
+let SpecFlowDefaults = { 
+    SubCommand =        "generateall"
+    ProjectFile =       null
+    ToolName =          toolname
+    ToolPath =          Tools.findToolFolderInSubPath toolname (currentDirectory </> "tools" </> "SpecFlow")
+    WorkingDir =        null
+    BinFolder =         null
+    OutputFile =        null
+    XmlTestResultFile = null
+    TestOutputFile =    null
+    Verbose =           false
+    ForceRegeneration = false
+    XsltFile =          null
+}
+
+// Runs SpecFlow on a project.
+/// ## Parameters
+///
+///  - `setParams` - Function used to manipulate the default SpecFlow parameter value.
+let generate setParams =    
+    let parameters = setParams SpecFlowDefaults
+
+    Trace.traceStartTaskUnsafe "SpecFlow " parameters.SubCommand
```

Comment:
We can use `use _ = Trace.traceStart ...` here to no longer be 'unsafe' :)

Reply:
Ok, thanks for the hint. Will fix that on the other PRs too